frontend/national-boxing-championships-explorer/style.css

```
@@ -0,0 +1,1608 @@
+  --boxing-navy: #f8f9fa;
+  --boxing-navy-light: #ffffff;
+  --boxing-text: #2c3e50;
+  --boxing-text-muted: #5d6d7e;
+  --boxing-glass: rgba(255, 255, 255, 0.85);
+  --boxing-border: rgba(192, 57, 43, 0.15);
+}
+
+/* ==========================================================================
+   Animation Keyframes
+   ========================================================================== */
+@keyframes fadeInUp {
```

Stylelint (17.14.0)

[error] Expected keyframe name "fadeInUp" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)